docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,220 @@
+---
+id: parallel-scraping-guide
+title: Parallel Scraping Guide
+sidebar_label: Parallel Scraping
+description: Parallelizing your scrapers with Crawlee
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+import SharedSource from '!!raw-loader!./shared.mjs';
+import AdaptedRoutesSource from '!!raw-loader!./adapted-routes.mjs';
+import ParallelScraperSource from '!!raw-loader!./parallel-scraper.mjs';
+import ModifiedDetailRouteSource from '!!raw-loader!./modified-detail-route.mjs';
+
+:::warning Experimental features ahead
+
+At the time of writing this guide (December 2023), request locking is still an experimental feature. You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
+
+:::
+
+In this guide, we will walk through how you can turn your single scraper into a scraper that can be parallelized and run in multiple instances. This guide
+assumes you've read and walked through our [introduction guide](../introduction/setting-up) (or have a fully-fledged scraper already built), but if you haven't done so yet,
+take a break, go read through all that, and come back. We'll be waiting...
+
+*Oh, you're back already! Let's proceed in making that scraper parallel!*
+
+## Things to consider before parallelizing
+
+Before you rush ahead and change your scraper to support parallelization, take a minute to consider the following factors:
+
+- do you plan on scraping so many pages that you need to parallelize your scraper?
+  > for example, if your scraper goes across few pages, you probably don't need parallelization  
+  > but if you scrape a lot of pages, or you scrape pages that take a long time to load, you might want to consider parallelization
+- can you parallelize your scraper while not overloading the source website?
+  > for example, if you scrape a website that has a lot of traffic, you don't want to add to that traffic by running multiple scrapers in parallel
+  > as that might cause the website to go down for all its users
+- do you have the resources available to run multiple scrapers in parallel?
+  > when running locally, depending on your scraper type, do you have enough CPU and RAM available to sustain multiple scrapers running in parallel  
+  > when running in the cloud, will the extra speed from parallelization be worth the extra cost of running multiple scrapers in parallel?
```

Comment:
```suggestion
- can you parallelize your scraper while not overloading the source website?
  - for example, if you scrape a website that has a lot of traffic, you don't want to add to that traffic by running multiple scrapers in parallel
  - as that might cause the website to go down for all its users
- do you have the resources available to run multiple scrapers in parallel?
  - when running locally, depending on your scraper type, do you have enough CPU and RAM available to sustain multiple scrapers running in parallel  
  - when running in the cloud, will the extra speed from parallelization be worth the extra cost of running multiple scrapers in parallel?
```

This way it would still be visually distinct,but it might look better? Especially the last question when you have 2 answers. They will be in separate bullet points instead of merged into one blockquote.

Reply:
Before and after

![Google Chrome - 2023-12-29 at 15 31 03@2x](https://github.com/apify/crawlee/assets/17960496/f219143f-27f7-4de6-a9b8-729169956f42)
![Google Chrome - 2023-12-29 at 15 31 46@2x](https://github.com/apify/crawlee/assets/17960496/ce2dc595-834d-409e-b026-79358ca19ceb)